source/CF/Constants.pm

```diff
@@ -267,7 +267,7 @@ sub list_clusterflow_genomes {
 
     my $returnstring = "";
 
-    my @config_files = ("$FindBin::Bin/genomes.config", "$homedir/.clusterflow/genomes.config", './genomes.config');
+    my @config_files = ("$FindBin::Bin/genomes.config","$FindBin::Bin/genomes.d/","$homedir/.clusterflow/genomes.config","$homedir/.clusterflow/genomes.d/",'./genomes.config');
```

Comment:
Does this need to be `glob("$FindBin::Bin/genomes.d/*config")` as above instead of just `"$FindBin::Bin/genomes.d/"`?